### PR TITLE
Add E2E test for private transactions

### PR DESCRIPTION
**Pull request by dmeijboom**
E2E test for private transactions. Build in two steps:

1. Create DIDs, NutsComm endpoints and add new DIDs to YAML config
2. Create NutsAuthorizationCredentials and assert if they are present on both nodes (thus the syncing works)

Closes: https://github.com/nuts-foundation/nuts-node/issues/713

### CODE REVIEW

File: nuts-network/private-transactions/prepare.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ../../util.sh
+
+function setupNode() {
+  local did=$(printf '{
+    "selfControl": true,
```
Comment (woutslakhorst) [inline nuts-network/private-transactions/prepare.sh:9]: don't they need a keyAgreement key for the encryption to work? Or is that one true by default? Then there's a mix of param settings and default settings?

File: nuts-network/private-transactions/run-test.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+# Wait for transactions to sync
+sleep 5
+
+if [ $(searchAuthCredentials "http://localhost:11323" | jq "length") -ne 2 ]; then
+  echo "failed to find NutsAuthorizationCredentials on Node-A"
+  exit 1
+fi
+
+if [ $(searchAuthCredentials "http://localhost:21323" | jq "length") -ne 2 ]; then
+  echo "failed to find NutsAuthorizationCredentials on Node-A"
```
Comment (woutslakhorst) [inline nuts-network/private-transactions/run-test.sh:69]: Node-B

**Review by woutslakhorst — CHANGES_REQUESTED** [review]

File: nuts-network/private-transactions/prepare.sh (hunk 1 above)
Comment (dmeijboom) [reply nuts-network/private-transactions/prepare.sh:9]: It's true by default. These are the default options:


```go
AssertionMethod:      true,
Authentication:       false,
CapabilityDelegation: false,
CapabilityInvocation: true,
KeyAgreement:         true,
SelfControl:          true,
```

Comment (dmeijboom) [reply nuts-network/private-transactions/prepare.sh:9]: Added it to the request to make it more clear

**Review by woutslakhorst — APPROVED** [review]

**Pull request merged**